src/components/AstronautsPage/AstronautsPage.tsx

```diff
@@ -1,10 +1,22 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { Container } from '@mui/material';
+import { useDispatch } from 'react-redux';
+import type { AppDispatch } from '../../store';
+import { getAstronautsData } from './actions/getAstronautsData';
 import { AstronautsList } from './components/AstronautsList';
 import './AstronautsPage.scss';
 
-export const AstronautsPage = () => (
-  <section className="astronauts-page">
-    <h1 className="astronauts-header">Astronauts</h1>
-    <AstronautsList />
-  </section>
-);
+export const AstronautsPage = () => {
+  const dispatch = useDispatch<AppDispatch>();
```

Comment:
You can't use AppDispath type in AstronautsPage

Reply:
i can use use _const useAppDispatch = () => useDispatch<AppDispatch>()_ construction like in documentation and add it to hooks.ts file or add it in store file exactly

Reply:
But you dispatch astronauts action not app

Reply:
changed on Dispatch type from redux with astronauts action